### ISSUE TITLE
BuilderSet import does not apply ElementNamingConvention

### ISSUE THREAD

**Issue by StrykerNinja** [HtmlTags/htmltags#80]
in an attempt to change the default element naming convention for my editors, I discovered that the BuilderSet import ignores any ElementNamingConvention changes.

In my Registry, I try
```
var htmlConventionLibrary = new HtmlConventionLibrary();
new DefaultAspNetMvcHtmlConventions().Apply(htmlConventionLibrary);
```

and the DefaultAspNetMvcHtmlConventions looks like
```
public class DefaultAspNetMvcHtmlConventions : HtmlConventionRegistry
{
public DefaultAspNetMvcHtmlConventions()
{
Library.TagLibrary.BuilderSetFor(ElementConstants.Editor, TagConstants.Default).NamingConvention(new DotNotationElementNamingConvention());
// ommited for brevity
}
}
```

Tracing back what the Apply() was doing led me to the BuilderSet Import method.

Note that
```
htmlConventionLibrary.TagLibrary.BuilderSetFor(ElementConstants.Editor, TagConstants.Default).NamingConvention(new DotNotationElementNamingConvention());
```
in the registry after the Apply will work, but I was hoping to use my convention class

**Comment by jbogard** [comment]
Oh that does indeed look like a bug. I'll take a look!

**Comment by jbogard** [comment]
Ah, I think I see the issue - the "Editors" property doesn't expose the underlying BuilderSet's method to apply a naming convention.

**Comment by StrykerNinja** [comment]
> Ah, I think I see the issue - the "Editors" property doesn't expose the underlying BuilderSet's method to apply a naming convention.

While true, you can still get to it via the TagLibrary. The underlying problem for me was that Since I am doing these things in side my custom `HtmlConventionRegistry`, the Apply method wasn't actually copying the changes (because it is eventually calling Import on the BuilderSet).

In other words, inside my 'HtmlConventionRegistry' When I would debug and look at the naming convention it was DotNamingConvention instead of Default, but when I would look again after the Apply, it had reverted.

**Comment by jbogard** [comment]
It's because when the BuilderSets are imported, this is missing:

```
public void Import(BuilderSet other)
{
    _policies.AddRange(other._policies);
    _modifiers.AddRange(other._modifiers);
    _elementNamingConvention = other._elementNamingConvention;
}
```

**Comment by jbogard** [comment]
OK I think this is fixed in  b721a3e which I'm marking 4.2.0 for the new support for MVC Core tag helpers.
